typegate/tests/runtimes/wasm/wasmedge_sync_test.ts

```diff
@@ -0,0 +1,156 @@
+// Copyright Metatype OÜ, licensed under the Elastic License 2.0.
+// SPDX-License-Identifier: Elastic-2.0
+
+import { BasicAuth, tgDeploy, tgRemove } from "@typegraph/sdk/tg_deploy.js";
+import { gql, Meta } from "test-utils/mod.ts";
+import { testDir } from "test-utils/dir.ts";
+import { tg } from "./wasmedge.ts";
```

Comment:
Don't forget to rename wasmedge references

Reply:
yeah, done now.